.github/workflows/build-test.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
Probably best to leave the switch from `master` to `main` to a separate PR. With a repo with this many PRs, that would be a fairly disruptive change.

Reply:
Good catch, updated to `main`

Reply:
*`master`